net.audiorelay.AudioRelay.yml

```diff
@@ -0,0 +1,38 @@
+app-id: net.audiorelay.AudioRelay
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: audiorelay
+
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --device=dri
+  - --persist=.skiko
+  - --persist=.java
+
+modules:
+  - name: audiorelay
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/audiorelay
+      - tar -xf audiorelay.tar.gz -C /app/audiorelay
+      - install -D -m644 /app/audiorelay/lib/AudioRelay.png /app/share/icons/hicolor/512x512/apps/net.audiorelay.AudioRelay.png
+      - install -D -m644 net.audiorelay.AudioRelay.metainfo.xml -t /app/share/metainfo
+      - install -D -m644 net.audiorelay.AudioRelay.desktop -t /app/share/applications
+      - install -D -m755 audiorelay -t /app/bin
+    sources:
+      - type: file
+        path: audiorelay
+      - type: file
+        path: net.audiorelay.AudioRelay.desktop
+      - type: file
+        path: net.audiorelay.AudioRelay.metainfo.xml
+      - type: file
+        dest-filename: audiorelay.tar.gz
+        only-arches:
+          - x86_64
```

Comment:
this isn't enough to restrict the build to x86_64.

Reply:
I have added the necessary things in `flathub.json`.